Makefile

```diff
@@ -368,10 +377,28 @@ vendor: ## Runs go mod vendor
 tidy: ## Runs go mod tidy
 	go mod tidy
 
-
 .PHONY:verify-vendor
 verify-vendor:tidy vendor verify-no-changes ##Verifies vendor and tidy didn't cause changes
 
-
 .PHONY:verify-bundle
-verify-bundle: manifests bundle verify-no-changes ##Verifies bundle and manifests didn't cause changes
+verify-bundle: manifests bundle bundle-reset verify-no-changes ##Verifies bundle and manifests didn't cause changes
+
+# Revert all version or build date related changes
+.PHONY: bundle-reset
+bundle-reset:
+	VERSION=0.0.1 $(MAKE) manifests bundle
+	# empty creation date
+	sed -r -i "s|createdAt: .*|createdAt: \"\"|;" ./bundle/manifests/$(OPERATOR_NAME).clusterserviceversion.yaml
+
+SORT_IMPORTS = $(shell pwd)/bin/sort-imports
+.PHONY: sort-imports
+sort-imports: ## Download sort-imports locally if necessary.
+	$(call go-install-tool,$(SORT_IMPORTS),github.com/slintes/sort-imports@$(SORT_IMPORTS_VERSION))
+
+.PHONY: test-imports
+test-imports: sort-imports ## Check for sorted imports
+	$(SORT_IMPORTS) .
+
+.PHONY: fix-imports
+fix-imports: sort-imports ## Sort imports
+	$(SORT_IMPORTS) . -w
```

Comment:
1. `make fix-imports` doesn't work with `. -w`, since it was changed in v0.2.1- ["requires flags to be before project dir now"](https://github.com/slintes/sort-imports/releases/tag/v0.2.1).
2. v0.2.1 does add few more sorting changes

Reply:
Hmm I don't quite follow.
As far as I see `make fix-imports` is executed ok and does what I expect it to do.
What is the change you suggest ?

Reply:
My mistake, was running with cached 0.1.0 version, fixed